src/newcharacter.cpp

```diff
@@ -997,6 +1012,9 @@ tab_direction set_stats(WINDOW *w, player *u, points_left &points)
         } else if (action == "NEXT_TAB") {
             delwin(w_description);
             return tab_direction::FORWARD;
+        } else if (action == "HELP_KEYBINDINGS") {
+            // Need to redraw since the help window obscured everything.
+            draw_tabs(w, _("STATS"));
         } else if (action == "QUIT" && query_yn(_("Return to main menu?"))) {
```

Comment:
> should I match the existing code style
That's the worst case.

> change my additions only, or change the style of the whole function?
Both of these cases are acceptable but in case of restyling the whole function it's better to do it as a separate commit.

All code in the repository (sooner or later) will be formatted using the official code style (described in ```doc```), so it's better to write your code in this style even if everything else around doesn't match it.